app/src/main/java/com/iic/mokojin/views/CharacterViewer.java

```diff
@@ -41,6 +68,25 @@ public CharacterViewer(Context context, AttributeSet attrs, int defStyleAttr) {
     private void init() {
         inflate(getContext(), R.layout.character_viewer, this);
         if (!isInEditMode()) ButterKnife.inject(this);
+        if (mSize == Size.small.ordinal()){
+            mFrontImage.setBorderColor(getResources().getColor(R.color.background_material_light));
+            
+            ViewGroup.LayoutParams layoutParams = mFrontImage.getLayoutParams();
+            layoutParams.height = getResources().getDimensionPixelSize(R.dimen.small_avatar_size);
+            layoutParams.width = getResources().getDimensionPixelSize(R.dimen.small_avatar_size);
+            mFrontImage.setLayoutParams(layoutParams);
+
+            ViewGroup.LayoutParams layoutParams2 = mBackImage.getLayoutParams();
+            layoutParams2.height = getResources().getDimensionPixelSize(R.dimen.small_avatar_size);
+            layoutParams2.width = getResources().getDimensionPixelSize(R.dimen.small_avatar_size);
+            mBackImage.setLayoutParams(layoutParams2);
+            
+            mMargin = getResources().getDimensionPixelSize(R.dimen.smaller_character_separation_amount);
+            mBorderWidth = getResources().getDimension(R.dimen.smaller_character_border_width);
+        } else {
+            mMargin = getResources().getDimensionPixelSize(R.dimen.character_separation_amount);
+            mBorderWidth = getResources().getDimension(R.dimen.character_border_width);
```

Comment:
I think we can hold the resources object instead of calling `getResources()` each time, no?